subete/repo.py

```diff
@@ -264,6 +264,13 @@ def _load_git_data(self) -> None:
         and inject that data into the repo object. This was done for simplicity.
         It seems like way more of a pain to try to pass the git data around.
         """
+
+        # Make sure .git-blame-ignore-revs exists for older versions of git and
+        # keep track of whether it existed before
+        blame_path = Path(f"{self._temp_dir.name}/.git-blame-ignore-revs")
+        blame_path_exists = blame_path.exists()
+        blame_path.touch()
```

Comment:
Had to do a bit of investigating with this line of code. I was worried it might overwrite an existing file, but it seems to work just fine.